client/src/components/shipping_info/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import "./style.css";
+import swal from "sweetalert";
+import NavBar from "../Navbar";
+import axios from "axios";
+
+class shippingInfo extends Component {
+  state = {
+    username: "",
+    phone: "",
+    Address: "",
+    Extra_Note: ""
+  };
+
+  changeInput = ({ target: { value, name } }) => {
+    this.setState({ [name]: value });
+  };
+
+  clickButton = () => {
+    const { username, phone, Address, Extra_Note } = this.state;
+    if (username !== "" && phone !== "" && Address !== "") {
+      axios
+        .post("/shipping_info", { username, phone, Address, Extra_Note })
+        .then(res => res.data);
+
+      swal({
+        title: "Your Order is processed Contact us on phone No : 0598121490",
+        icon: "success",
+        button: "Home!"
+      }).then(function() {
+        window.location.href = "/";
+      });
```

Comment:
The Sweet alert confirmation message will appear here even the form didn't send the data to the backend.
you should check first if the form data is already sent to the backend after that let the sweet alert message appear.